patterns/about-details-description-with-bullets.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Title: About Details Description With Bullets
+ * Slug: twentytwentyfive/about-details-description-with-bullets
+ * Categories: text, about, columns
+ * Description: About Details Description With Bullet.
```

Comment:
Having the title and the description be the same text is not helpful.
From these texts, I can't understand what the pattern does.